src/core/ComboBox/ComboBox.tsx

```diff
@@ -30,6 +31,14 @@ export type ComboBoxProps<T> = {
    * Controlled value of ComboBox.
    */
   value?: T;
+  /**
+   * Message below the select.
+   */
+  message?: React.ReactNode;
```

Comment:
Since we can't seem to agree on a name for the icon prop, I'm going to suggest merging the icon prop into the message by offering a new component. Usage could look something like this:
```tsx
<ComboBox status='negative' message={<Message>Oh noo</Message>} />
```
```tsx
<ComboBox
  message={<Message icon={SvgCamera}>This is a camera for some reason!</Message>}
  // ...
/>
```

With some CSS changes, it could also be used in other places (like a table cell):
```tsx
<Message status='negative'>This row failed!</Message>
```

The name of the new component is debatable.